src/main/java/de/infynyty/zuap/Zuap.java

```diff
@@ -58,6 +61,10 @@ private static void parseWebsiteData(final InsertionAnnouncer announcer){
         });
     }
 
+    public static boolean getNoWGZimmer() {
```

Comment:
Should be `isWGZimmerEnabled()` instead, see the other comment for more details.

Reply:
done